test_ped.py

```diff
@@ -8,6 +8,14 @@
 
 import ped
 from ped.guess_module import guess_module
+from pathlib import Path
+
+
+def test_dir_opening(monkeypatch):
+    with monkeypatch.context() as m:
```

Comment:
I think this is unnecessary; can you try removing this?

Reply:
That's what I'm using to set the env.

Reply:
Or, you just don't want it invoked as a context manager?

Reply:
Yeah, I meant that the context manager is unnecessary. I think you can just do `monkeypatch.setenv`.